src/index.js

```diff
@@ -12,6 +12,7 @@ import Default from './routes/Default';
 import SideBar from './components/SideBar';
 import { ID_TOKEN_KEY } from './common/constants';
 import { theme } from './styles/theme';
+import '@trendmicro/react-toggle-switch/dist/react-toggle-switch.css';
```

Comment:
Let's import the stylesheet for the toggle switch only in the components that use the toggle switch.